src/main/java/edu/ucsb/cs156/example/documents/Metadata.java

```diff
@@ -0,0 +1,29 @@
+package edu.ucsb.cs156.example.documents;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import org.springframework.data.annotation.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Metadata {
+    @Id
+    private String _id;
+
+    private int generated;
```

Comment:
The description for the metadata says: "but you can see that it would simply be private String fields for everything here except for the count, which would be a private int."
Though in the example it looks like there are 3 ints and not just 1. Not sure which one is the correct interpretation though.

Reply:
I used the one that's listed on the USGS response